web/styles/popovers.css

```diff
@@ -615,6 +619,30 @@ ul {
             }
         }
     }
+
+    /* The height of the main body of the profile modal is 60vh. However, */
```

Comment:
Good comment here. Two things:

1. You should probably put this one up by where you specify `height: 52vh`.
2. You can just open the CSS comment on the first line with `/*, and then indent subsequent lines by three spaces, and then close the comment at the end of the last line after a space.

Reply:
Changed it, thanks for the suggestion